packages/react-openapi/src/OpenAPIContextProvider.tsx

```diff
@@ -0,0 +1,72 @@
+'use client';
+
+import * as React from 'react';
+import { OpenAPIOperationData } from './fetchOpenAPIOperation';
+import { getServersURL } from './utils';
+
+type OpenAPIContextProps = {
+    isPending?: boolean;
+    serverUrl?: string;
+    state?: Record<string, string> | null;
+    onUpdate: (params: Record<string, string> | null) => void;
+};
```

Comment:
We should document these props, I have no idea what they mean (`isPending`, etc).